project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/elastisch "3.0.0-beta2-SNAPSHOT"
+(defproject clojurewerkz/elastisch "5.0.0-alpha1-SNAPSHOT"
```

Comment:
Since there will be no major Elastisch release for every ElasticSearch release in practice, the argument for matching versions only goes so far. But I've renamed the `4.0` milestone to `5.0`. It costs us nothing to jump a version once.